tests/conftest.py

```diff
@@ -26,3 +29,22 @@ def flask_test_client():
     """
     with create_app().test_client() as test_client:
         yield test_client
+
+
+@pytest.fixture
+def seed_test_dataset(flask_test_client):
+    """
+    Pushes an application context to a test.
+
+    :return: a test application context.
+    """
+    with flask_test_client.application.app_context():
+        seed_test_database(Organisation, db_seed.ORGANISATION_DATA)
+        seed_test_database(Contact, db_seed.CONTACT_DATA)
+        seed_test_database(Package, db_seed.PACKAGE_DATA)
```

Comment:
I propose that we could pass the `app_ctx` fixture to this function to reuse it and also rename this fixture to `seeded_app_ctx` and update the docstring

Reply:
i.e.

```
@pytest.fixture
def seeded_app_ctx(app_ctx):
    """
    Updated docstring
    """
    seed_test_database(Organisation, db_seed.ORGANISATION_DATA)
    seed_test_database(Contact, db_seed.CONTACT_DATA)
    seed_test_database(Package, db_seed.PACKAGE_DATA)
    db.session.commit()
    yield
```

Reply:
Nice, the re-ordering makes sense. Yeah docstring looks all g